data/showcase/impacts_budget.yml

```diff
@@ -0,0 +1,12 @@
+title: Impacts du budget
+description:
+  fr: Exploration interactive de l'impact des projets de loi de finance et loi de finance de la sécurité sociale sur le budget de l'État et le niveau de vie des ménages.
+  en: ??
```

Comment:
Interactive exploration of the impact of recent fiscal reforms on public revenues and inequality